projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -41,8 +43,20 @@ export const withAIAssistant = createHigherOrderComponent(
 
 		const clientIdsRef = useRef< Array< string > >();
 
+		const { name: blockType } = props;
+
 		const { updateBlockAttributes, removeBlocks } = useDispatch( blockEditorStore );
 		const { createNotice } = useDispatch( noticesStore );
+		const { replaceBlock } = useDispatch( blockEditorStore );
```

Comment:
You could extract `replaceBlock` from line `48`, not? It uses `useDispatch( blockEditorStore )` too.

Reply:
Done in https://github.com/Automattic/jetpack/pull/31568/commits/2528359cbba3441b1ae46f5a03b2153438a8d1dd